src/i18n/langs/en.json

```diff
@@ -28,8 +27,22 @@
   "Copyright": "Copyright",
   "Page Not Found": "Page Not Found",
   "Approved by the International Year of Basic Sciences for Sustainable Development 2022": "Approved by the International Year of Basic Sciences for Sustainable Development 2022",
-  "Energy balance": "Energy balance",
-  "Global model including radiation, greenhouse gases, clouds and ice": "Global model including radiation, greenhouse gases, clouds and ice",
+  "GLOBAL_MODEL_TITLE": "Global Model",
+  "GLOBAL_MODEL_DESCRIPTION": "Radiative balance with greenhouse gases, clouds and surface",
   "Based on climate education research • Adapted for middle and high schools • Tested in classrooms • Easily embedded in Graasp activities and other web platforms": "Based on climate education research • Adapted for middle and high schools • Tested in classrooms • Easily embedded in Graasp activities and other web platforms",
-  "Brief tutorial of the simulations": "Brief tutorial of the simulations"
+  "Brief tutorial of the simulations": "Brief tutorial of the simulations",
+  "CHEMISTRY_SIMULATIONS_TITLE": "Chemistry Simulations",
+  "SEND_EMAIL_BUTTON": "Send us an email",
+  "CONTACT_FORM_TEXT": "We'll be more than happy to hear about your suggestion or feedback regarding your use of the simulations. It will help for further improvements. Write an email to us, we will reply in a few days.",
```

Comment:
We would love to hear your suggestions, comments, and feedback on the simulations.